spec/support/chargeback_helper.rb

```diff
@@ -10,6 +10,18 @@ def set_tier_param_for(metric, param, value, num_of_tier = 0)
       def used_average_for(metric, hours_in_interval, resource)
         resource.metric_rollups.sum(&metric) / hours_in_interval
       end
+
+      def add_metric_rollups_for(resources, range, step, custom_params, trait = :with_data)
```

Comment:
thanks @chrisarcand, updated 👍 
1. Ok, will add when it will be needed.
2. 👍 
3. 👍